pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1115,6 +1115,10 @@ func PublishDeviceInfoToZedCloud(ctx *zedagentContext) {
 
 	ReportDeviceInfo.RestartCounter = ctx.restartCounter
 
+	//Operational information about TPM presence/absence/usage.
+	//"Absent" for now, till we enable TPM functionality.
+	ReportDeviceInfo.HwSecurityModule = "Absent"
```

Comment:
Presumably "Unknown" makes more sense initially.

Reply:
@nordmark Updated the PR, please have a look.